chain/actors/manifest.go

```diff
@@ -140,3 +129,19 @@ func CanonicalName(name string) string {
 
 	return name
 }
+
+func GetBuiltinActorsKeys() []string {
```

Comment:
Let's put this at the top of the file (right after the keys themselves are defined).